### ISSUE TITLE
Instead of loading, just throws a bunch of toasts with URLs

### ISSUE THREAD

**Issue by gappleto97** [stom79/mastalab#433]
The most recent update on FDroid seems to be badly broken.

**Comment by cbondurant** [comment]
I observed the same problem, screenshotted one of the toasts that appeared. Seems to be some form of rate limiting issue? 
![screenshot_20180430-194310](https://user-images.githubusercontent.com/12928246/39455540-fa96346e-4cae-11e8-99e8-f0822b9a8e49.png)

**Comment by cbondurant** [comment]
after looking into this a bit more, it seems that this is actually an error on mastodon.social's part, as the site itself is throwing 500 errors. @gappleto97 is this the instance you are attempting to connect to as well?

**Comment by gappleto97** [comment]
Yes. In that case, im going to guess it isn't the app's fault, though it would be nice if it had better error handling.
